src/preferences/node/preference-server-module.ts

```diff
@@ -26,17 +24,7 @@ export const preferenceServerModule = new ContainerModule(bind => {
     bind(WorkspacePreferenceServer).to(JsonPreferenceServer).inSingletonScope();
     bind(UserPreferenceServer).to(JsonPreferenceServer).inSingletonScope();
 
-    // bind(JsonPreferenceServer).to(WorkspacePreferenceServer).inSingletonScope();
-    // bind(PreferencePath).toConstantValue(path.join('.theia', 'prefs.json')).whenInjectedInto(WorkspacePreferenceServer.toString());
-    // const userHome = ".default"; // FIXME
-    // bind(PreferencePath).toConstantValue(path.join(userHome, '.theia', 'prefs.json')).whenInjectedInto(UserPreferenceServer.toString());
-    // const home = ctx.container.get(FileSystemNode);
-
-    const root = getRootDir();
-    if (root) {
-        bind(PreferencePath).toConstantValue(FileUri.create(path.resolve(root, '.theia', 'prefs.json')));
-    }
-
+    bind(PreferencePath).toConstantValue(FileUri.create(path.resolve('.theia', 'prefs.json')));
```

Comment:
it should be `workspaceServer.getRoot().then(root => root.resolve(path.resolve('.theia', 'prefs.json'))`